packages/manager/modules/pci/src/projects/project/analytics-data-platform/deploy/quota-error-model/quota-error-model.html

```diff
@@ -0,0 +1,59 @@
+<oui-modal data-title="{{ ::'analytics_data_platform_deploy_quota_error_model_title' | translate }}"
+           data-primary-action="$ctrl.closeModal()"
+           data-primary-label="{{ ::'analytics_data_platform_deploy_quota_error_model_button_increase_quota' | translate }}"
+           data-secondary-action="$ctrl.dismissModal()"
+           data-on-dismiss="$ctrl.dismissModal()"
+           data-secondary-label="{{ ::'analytics_data_platform_common_close' | translate }}"
+           data-type="error"
+           data-ng-init="localizedStorageUnit = ('analytics_data_platform_common_unit_gb' | translate)">
+    <p data-translate="analytics_data_platform_deploy_quota_error_model_description"
+       data-translate-values="{cloud_name: $ctrl.publicCloudName}">
+    </p>
+    <div class="oui-datagrid-responsive-container
```

Comment:
Any reason not to use `oui-datagrid` ?

Reply:
This is a simple table with no pagination, sorting, filter, search. Number of rows and columns are fixed. Also we need row row header feature which is not supported in datagrid.